docs/_data/icons.json5

```diff
@@ -144,8 +144,12 @@
         ],
     },
     {
-        name: 'flag',
-        minver: '0.7.0',
+      name: 'fish-hook',
+      minver: '0.25.0',
```

Comment:
@araiford, July was our last pre-v1.0 minor release.  

Please change to:

```json
minver: '1.0.0',
```

Reply:
Thanks James!

Done